IO_Demosaic/IO_Demosaic.cs

```diff
@@ -9,7 +9,7 @@ namespace IO_Demosaic
     public class IO_Demosaic : BaseUnityPlugin
     {
         // Set version in BepInEx and in AssemblyInfo
-        public const string Version = "1.1";
+        public const string Version = "1.2";
```

Comment:
```suggestion
        public const string Version = "1.1";
```
Do not change version numbers in PRs.